app/controllers/users_controller.rb

```diff
@@ -166,6 +166,10 @@ def edit_user_params
     params.require(:user).permit(:email, :first_name, :last_name)
```

Comment:
Do we need to conditionally restrict them? Not sure quite what the best way is to do this.